index.html

```diff
@@ -66,11 +66,17 @@ <h3>Settings</h3>
       </div>
     </form>
     <div class="buttons">
-      <button id="resetBtn" class="action">Restore defaults</button>
-      <button id="submitBtn" class="action">Apply</button>
+      <button id="resetBtn" class="action" onclick="resetDefaultSettings()">Restore defaults</button>
+      <button id="submitBtn" class="action" onclick="submitSettings()">Apply</button>
```

Comment:
```suggestion
      <button id="resetBtn" class="action">Restore defaults</button>
      <button id="submitBtn" class="action">Apply</button>
```
The onclick event has been mapped in the JS now, see app.mjs line 219.